android/build.gradle

```diff
@@ -30,4 +33,29 @@ android {
     lintOptions {
         disable 'InvalidPackage'
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+
+    dependencies {
+        repositories {
+            mavenCentral()
+            maven { url "https://dl.bintray.com/terl/lazysodium-maven" }
+        }
+
+        implementation ('com.tozny.e3db:e3db-client-android:5.0.1@aar') {
```

Comment:
Is there a reason this isn't using 6.0.0?

Reply:
Nope will move to 6.0.0 for identity features